spyder/plugins/ipythonconsole/utils/kernelspec.py

```diff
@@ -62,13 +94,41 @@ def argv(self):
                 pyexec = pyexec_w
 
         # Command used to start kernels
-        kernel_cmd = [
-            pyexec,
-            '-m',
-            'spyder_kernels.console',
-            '-f',
-            '{connection_file}'
-        ]
+        if is_different_interpreter and is_conda_env(pyexec=pyexec):
+            # If this is a conda environment we need to call an intermediate
+            # activation script to correctly activate the spyder-kernel
+
+            # If changes are needed on this section make sure you also update
+            # the activation scripts at spyder/plugins/ipythonconsole/scripts/
+            if os.name == 'nt':
+                import ctypes
+                codepage = str(ctypes.cdll.kernel32.GetACP())
+                kernel_cmd = [
+                    get_activation_script(),  # This is bundled with Spyder
+                    codepage,
+                    get_conda_activation_script(pyexec),  # Might be remote
+                    get_conda_env_path(pyexec),  # Might be remote
```

Comment:
remote -> external

Remote is a different thing.

Reply:
Fixed